university-college-lillebaelt-vancouver.csl

```diff
@@ -270,7 +278,7 @@
   </citation>
   <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
     <layout>
-      <text variable="citation-number" suffix="."/>
+      <text variable="citation-number" suffix=". "/>
```

Comment:
As the style is now (without the space), if you insert a bibliography with
only a few references, and later add more, the bibliography will look like
this after the tenth citation:


[image: Inline image 2]
I was rather puzzled about this, especially because the standard Vancouver
style (the one that Zotero is 'born' with) doesn't have this problem. So I
checked how that csl was written, and noticed that the citation number had
". " as suffix rather than just "."

I guess the real problem is, that the size of the hanging indent is decided
when the bibliography is made, and not adjusted later. Adding a space is an
unelegant fix, and it probably won't work if there is more than 99
citations, but the style is meant for bachelor students, and it is very
rare that they reach that kind of citation count in their papers.

I noticed that if you change citation style in your document, and then
change it back to Vancouver (or any other style with
second-field-align=flush I guess) the bibliography will realign. It's what
I would do, if I just used the style for myself. But I'm trying to 'sell'
Zotero to our students, and want it to work as seamlessly as possible for
them.

On Wed, Nov 11, 2015 at 4:07 PM, Rintze M. Zelle <notifications@github.com>
wrote:

> In university-college-lillebaelt-vancouver.csl
> <https://github.com/citation-style-language/styles/pull/1754#discussion_r44541527>
> :
>
> > @@ -270,7 +278,7 @@
> >    </citation>
> >    <bibliography et-al-min="7" et-al-use-first="6" second-field-align="flush">
> >      <layout>
> > -      <text variable="citation-number" suffix="."/>
> > +      <text variable="citation-number" suffix=". "/>
>
> No, makes sense. Just making sure I follow your logic.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/citation-style-language/styles/pull/1754/files#r44541527>
> .
>